src/imp/platform/browser/options_parser.js

```diff
@@ -54,6 +54,11 @@ module.exports.parseScriptElementOptions = function (opts, browserOpts) {
         opts.component_name = componentName;
     }
 
+    let userId = dataset.user_id;
+    if (userId) {
+        opts.default_span_tags = {'join:user_id': userId};
```

Comment:
Don't forget to run `make lint` as well. It's nitty and will complain about the whitespace here.